app/components/popups/textdialog.py

```diff
@@ -0,0 +1,78 @@
+import cutlet
+import argostranslate.package as package
+import argostranslate.translate
+
+from PyQt5.QtWidgets import QLabel, QTextEdit, QVBoxLayout, QDialog
+from PyQt5 import QtCore
+
+class TextDialog(QDialog):
+    """
+    This "window" is a QWidget. If it has no parent, it
+    will appear as a free-floating window as we want.
+    """
+    def __init__(self):
+        super().__init__()
+        layout = QVBoxLayout()
+
+        self.label_ocr = QLabel("OCR")
+        layout.addWidget(self.label_ocr)
+
+        self.line_edit_ocr = QTextEdit("")
+        layout.addWidget(self.line_edit_ocr)
+
+        self.label_romanji = QLabel("Romanji")
+        layout.addWidget(self.label_romanji)
+
+        self.line_edit_romanji = QTextEdit("")
+        layout.addWidget(self.line_edit_romanji)
+
+        self.label_translation= QLabel("Translation")
+        layout.addWidget(self.label_translation)
+
+        self.line_edit_translation = QTextEdit("")
+        layout.addWidget(self.line_edit_translation)
+
+        self.setLayout(layout)
+        self.setWindowFlags(QtCore.Qt.WindowStaysOnTopHint)
+        self.setAttribute(QtCore.Qt.WA_X11NetWmWindowTypeUtility)
+        self.resize(500, 200)
+
+
+        self.text = ""
+        self.ktkn2romanji = cutlet.Cutlet()
```

Comment:
From what I understand, the package seems to format japanese text. Use the full name `katakanaToRomaji` or `jpnFormatter`